directus-remix/index.md

```diff
@@ -0,0 +1,331 @@
+      fields: ["slug", "title", "published_date", { author: ["name"] }],
+      sort: ["-published_date"],
+    })
+  );
+  return { posts };
+};
+
+
+
+```
+
+This query will retrieve the first 100 items (default), sorted by published date (descending order, which is latest first). It will only return the specific fields we request - `slug`, `title`, `published_date`, and the `name` from the related `author` item.
+
+Display the fetched data in HTML:
+
+```tsx
+export default function Blog() {
+  const { posts } = useLoaderData<typeof loader>();
+  return (
+    <div>
+      <h1>Blog Posts</h1>
+      <ul>
+        {posts.map((post) => {
+          return (
+            <li key={post.slug}>
+              <a href={`/blog/${post.slug}`}>
+                <h2>{post.title}</h2>
+              </a>
+              <span>
+                {post.published_date} &bull; {post.author.name}
+              </span>
+            </li>
+          );
+        })}
+      </ul>
+    </div>
+  );
+}
+
+```
+
+Visit `http://localhost:3000/blog` and you'll find a blog post listing, with the latest items first.
+
+ ![A page with a title of "Blog". On it is a list of three items - each with a title, author, and date. The title is a link](blog-post.png)
+
+### Create Blog Post Pages
+
+Each blog post links to a page that does not yet exist. In the `app/routes` directory, create a new file called `blog.$slug.tsx` with the content:
+
+```tsx
+import { LoaderFunctionArgs } from "@remix-run/node";
+import directus from "../lib/directus";
+import { readItem } from "@directus/sdk";
+import { useLoaderData } from "@remix-run/react";
+
+
+export const loader = async ({ params }: LoaderFunctionArgs) => {
+  const { slug } = params;
+  const post = await directus.request(readItem("posts", slug as string));
+  return post;
+};
+
+export default function Post() {
+  const { image, title, content } = useLoaderData<typeof loader>();
+  return (
+    <div>
+      <img src={`${directus.url}assets/${image}?width=600`} alt="" />
+      <h1>{title}</h1>
+      <div dangerouslySetInnerHTML={{ __html: content }}></div>
+    </div>
+  );
+}
+
+
+```
+
+Some key notes about this code snippet.
+
+- The `width` attribute demonstrates Directus' built-in image transformations.
+- Once again, `dangerouslySetInnerHTML={{ __html: content }}` should only be used if all content is trusted.
+- Because almost-all fields are used in this page, including those from the image relational field, the `fields` property when using the Directus JavaScript SDK can be set to `*.*`.
+
+Click on any of the blog post links, and it will take you to a blog post page complete with a header image.
+
+![A blog post page shows an image, a title, and a number of paragraphs.](blog-single.png)
+
+## Add Navigation
+
+While not strictly Directus-related, there are now several pages that aren't linked to each other. Update the `Layout.astro` file to include a navigation. Don't forget to use your specific page slugs.
```

Comment:
Again, this says Astro. Copying and pasting is a convenience but has to come with the due diligence that work is updated for a specific use case.